src/Sulu/Bundle/MediaBundle/Resources/js/components/MediaSelectionItem/MediaSelectionItem.js

```diff
@@ -3,28 +3,47 @@ import React from 'react';
 import MimeTypeIndicator from '../MimeTypeIndicator';
 import mediaSelectionItemStyle from './mediaSelectionItem.scss';
 
-type Props = {
+type Props = {|
     mimeType: string,
     children: string,
     thumbnail?: string,
-};
+    thumbnailSize: number,
+    thumbnailMargin: number,
```

Comment:
Usually we don't pass numeric style values like this... E.g. in the `Overlay` component we have a `size` prop, which can takes values like `small` or `large`. That avoids inline styles and wouldn't allow for mixed styles all over the place.

In general I am wondering if it makes sense to extract that component... Doesn't seem to extract a lot of code.